scenes/game_elements/props/checkpoint/components/checkpoint.gd

```diff
@@ -1,17 +1,27 @@
 # SPDX-FileCopyrightText: The Threadbare Authors
 # SPDX-License-Identifier: MPL-2.0
-class_name Checkpoint extends Area2D
-
+class_name Checkpoint
+extends Area2D
 ## A place where the player respawns if the current scene is reloaded.
 
 ## The point where the player will spawn.
 @onready var spawn_point: SpawnPoint = %SpawnPoint
 
+## The sprite displayed when this checkpoint is activated
+@onready var sprite: Sprite2D = %Sprite
+
+## The collision shape for the sprite, when this checkpoint is activated
+@onready var collision_shape: CollisionShape2D = %CollisionShape
+
 
 func _ready() -> void:
+	sprite.visible = false
+	collision_shape.set_deferred(&"disabled", true)
```

Comment:
Why not disable the collision shape in the saved scene?

Reply:
Good question... I think I wanted it to be visible in the scene.

Reply:
I adjusted this to set it to disabled in the scene.